Gemfile

```diff
@@ -32,7 +33,7 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem 'quiet_assets', '~> 1.1.0'
-  gem 'govuk-lint', '0.5.0'
+  gem 'govuk-lint'
```

Comment:
Those are preferences I believe. Having it pinned in the Gemfile we will know the version by looking at the Gemfile and will not require us to look at the Gemfile.lock file. 

I have no strong preference, but it seems that we prefer to pin down gem versions in this project.